src/ui/win32/bindings/MemoryViewerControlBinding.cpp

```diff
@@ -34,7 +35,10 @@ INT_PTR CALLBACK MemoryViewerControlBinding::WndProc(HWND hControl, UINT uMsg, W
             return FALSE;
 
         case WM_LBUTTONUP:
-            OnClick({ GET_X_LPARAM(lParam) - MEMVIEW_MARGIN, GET_Y_LPARAM(lParam) - MEMVIEW_MARGIN });
+            if (bShiftHeld)
```

Comment:
if `bShiftHeld` is only needed for this case, recommend only evaluating it here.
```
if (GetKeyState(VK_SHIFT) < 0)
```